app/src/main/res/values/booleans.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <bool name="isTablet">false</bool>
+</resources>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Done